doc/architecture/software-update.rst

```diff
@@ -35,7 +63,7 @@ and consumed by the software update client running on supported devices.
 
 Update information is secured by generating hashes of each file and then
 signing the file containing the references to the hashes.
-A secure channel is not required to distribute the updates.
+There is no need for a secure channel, for distributing the updates.
```

Comment:
Let's say that I was too optimistic about the feature being available on time.
I will amend the doc.